File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex23.java
//Importando Scanner
import java.util.Scanner;


public class Ex23 {

	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
	
	//vars
	int angulo1, angulo2, angulo3;
	
	
	System.out.println("Medida do 3° ângulo de um triângulo\n\n");
	
	//Entradas	
	System.out.print("Insira o tamanho do 1° ângulo: ");
	angulo1 = ent.nextInt();
	
	System.out.print("Insira o tamanho do 2° ângulo: ");
	angulo2 = ent.nextInt();
	
	//Processamento
	angulo3 = 180 - angulo2 - angulo1;
	
	//Saída
	System.out.println("3° ângulo = " + angulo3 + "°");
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex23.c
#include <stdio.h>

main()
{
    //vars
    int angulo1, angulo2, angulo3;

    printf("\t\t\t3o angulo de um triangulo\n\n");

    //Entrada
    printf("Insira o valor do 1o angulo: ");
    scanf("%d",&angulo1);

    printf("Insira o valor do 2o angulo: ");
    scanf("%d",&angulo2);

    //Processamento
    angulo3 = 180 - angulo1 - angulo2;

    //Saída
    printf("3o angulo = %i", angulo3);

}

